### PR TITLE
Update content to curriculum/Web/HTML/images

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/images/image-base.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/images/image-base.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/images/image-base.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/images/image-base.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - Web/HTML/images/image-title.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md
  - Web/HTML/images/image-title.md

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - Web/HTML/images/image-title.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md
  - Web/HTML/images/image-title.md
  - Web/HTML/images/image-types.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md
  - Web/HTML/images/image-types.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-base.md
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md
  - Web/HTML/images/image-types.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md
  - Web/HTML/images/image-types.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-map.md
  - Web/HTML/images/image-types.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md
  - Web/HTML/images/image-types.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/images/image-alt.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/images/image-height-width.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-alt.md?ref=6138da70f4dd36a7caecd0040294d63b87edba3f
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-base.md?ref=6138da70f4dd36a7caecd0040294d63b87edba3f
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-height-width.md?ref=6138da70f4dd36a7caecd0040294d63b87edba3f
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-map.md?ref=6138da70f4dd36a7caecd0040294d63b87edba3f
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-title.md?ref=6138da70f4dd36a7caecd0040294d63b87edba3f
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-types.md?ref=6138da70f4dd36a7caecd0040294d63b87edba3f

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-alt.md?ref=8f543b14bd1c23861bec9ef6e9f78dc6acba7b50
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-base.md?ref=8f543b14bd1c23861bec9ef6e9f78dc6acba7b50
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-height-width.md?ref=8f543b14bd1c23861bec9ef6e9f78dc6acba7b50
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-map.md?ref=8f543b14bd1c23861bec9ef6e9f78dc6acba7b50
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-title.md?ref=8f543b14bd1c23861bec9ef6e9f78dc6acba7b50
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-types.md?ref=8f543b14bd1c23861bec9ef6e9f78dc6acba7b50

File: Web/HTML/images/image-types.md [hunk 1]
```diff
@@ -84,7 +124,7 @@ Which reasons is web image compression important?
 - Prevents images from being stolen 
 - Security from deleting the image
 
-Are GIF images are the only image format that can be animated using CSS on a web page?
+Are GIF images the only image format that can be animated using CSS on a web page?
```
Comment (lizTheDeveloper) [inline Web/HTML/images/image-types.md]: can only have one question in a quiz section

File: Web/HTML/images/image-map.md [hunk 2]
```diff
@@ -66,14 +199,20 @@ When using the map element to create clickable areas within a single image, what
...
-
+### How much do you know about create clickable image maps?
 Place the shape values within the appropriate locations to create the proper code.
 
-```<map name="objects">
-  <area shape="???" coords="10,50,82,126" alt="" href="object1.htm">
-  <area shape="???" coords="90,58,3" alt="" href="object2.htm">
-  <area shape="???" coords="19,24,12,32,25,41,34,33,32,25" alt="" href="object3.htm">
+```
+<map name="objects">
```
Comment (lizTheDeveloper) [inline Web/HTML/images/image-map.md]: Can only have one ??? in a quiz type

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-alt.md?ref=6db95fa332387b66d65665b19afe47430698d01b
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-base.md?ref=6db95fa332387b66d65665b19afe47430698d01b
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-height-width.md?ref=6db95fa332387b66d65665b19afe47430698d01b
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-map.md?ref=6db95fa332387b66d65665b19afe47430698d01b
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-title.md?ref=6db95fa332387b66d65665b19afe47430698d01b
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-types.md?ref=6db95fa332387b66d65665b19afe47430698d01b

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-alt.md?ref=e8a6149ccdd94a02eba8fba3c3081874bc1e8c48
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-base.md?ref=e8a6149ccdd94a02eba8fba3c3081874bc1e8c48
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-height-width.md?ref=e8a6149ccdd94a02eba8fba3c3081874bc1e8c48
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-map.md?ref=e8a6149ccdd94a02eba8fba3c3081874bc1e8c48
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-title.md?ref=e8a6149ccdd94a02eba8fba3c3081874bc1e8c48
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/Web/HTML/images/image-types.md?ref=e8a6149ccdd94a02eba8fba3c3081874bc1e8c48

**Comment by lizTheDeveloper** [comment]
This has been migrated to #766

**Pull request closed without merging**